Components/Sensors/BME280.cs

```diff
@@ -0,0 +1,367 @@
+﻿using System;
+using Scarlet.IO;
+using Scarlet.Utilities;
+
+namespace Scarlet.Components.Sensors
+{
+    /// <summary>
+    /// Bosch BME280 - Integrated Air Humidity, Pressure and Temperature Sensor
+    /// Datasheet v1.1: https://download.mikroe.com/documents/datasheets/BST-BME280_DS001-11.pdf
+    /// Datasheet v1.19: https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMP280-DS001-19.pdf
+    ///     NOTE: This is for a similar device, the BMP280 (P for pressure only). This describes register LSB/MSB order, but has no info about humidity.
```

Comment:
Do you mean: "This is _not_ for"?

Reply:
Nope, the comment is correct. It's talking about that datasheet link, which is for the BMP280, but has some info that is necessary to understand how the BME280 (this device) works.

Reply:
Ah, that makes sense and would answer why it's tabbed in